core/modules/parser/QSMySqlListener.cc

```diff
@@ -378,18 +395,27 @@ class RootAdapter :
         public Adapter,
         public DmlStatementCBH {
 public:
-    RootAdapter() {}
+    RootAdapter()
+    : _ctx(nullptr)
+    {}
 
     shared_ptr<query::SelectStmt>& getSelectStatement() { return _selectStatement; }
 
     void handleDmlStatement(shared_ptr<query::SelectStmt>& selectStatement) override {
         _selectStatement = selectStatement;
     }
 
-    void onExit() override {}
+    void onEnter(QSMySqlParser::RootContext * ctx) {
```

Comment:
Extra space in 'RootContext *'?

Reply:
I think I was following a pattern from elsewhere in the qserv/query code. It's definitely not my instinct to put spaces on either side, but it *is* my instinct to make my code look like other code around me. But I'm not sure why I did it here....